src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  return comments.map(comment => (
+    <div className="CommentList">
+      <CommentInfo comment={comment} />
+    </div>
+  ));
```

Comment:
When rendering a list of elements in React, each element should have a unique `key` prop to help React identify which items have changed, are added, or are removed. Consider adding a `key` prop to the `div` element, using a unique identifier from the `comment` object, such as `comment.id`.